scripts/release_signal.py

```diff
@@ -0,0 +1,10 @@
+import os
+from subprocess import CalledProcessError, check_output
+
+try:
+    check_output('git describe --exact-match HEAD'.split())
+    print("We are in a tagged branch, let's do the release")
+except CalledProcessError as e:
+    print("We are not in a tagged branch")
+else:
```

Comment:
I feel it is the correct structure to put here... I try to catch the exception... if the exception is not raised, then I add the env variable... putting the action inside the try block feels unnatural to me because I am not trying to catch the exception on that "action", instead I am trying to catch the exception in the check_output action...